dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_submissions_controller.rb

```diff
@@ -1,11 +1,12 @@
 class Api::V1::Pd::WorkshopSurveyFoormSubmissionsController < ApplicationController
+  include Pd::WorkshopSurveyFoormConstants
   def create
     answers = params[:answers]
 
     # save facilitator answers as separate survey submissions
     # for ease of querying per-facilitator data
-    facilitator_answers = answers["facilitators"]
-    answers.delete("facilitators")
+    facilitator_answers = answers[FACILITATORS]
```

Comment:
Would it be slightly clearer where `FACILITATORS` came from if it had a namespace or some other kind of prefix?

Reply:
I can prefix with Pd::WorkshopSurveyFoormConstants if that makes it clearer. Should there be a prefix on the variable name itself instead?

Reply:
I think the namespace might be helpful enough.